babel.config.js

```diff
@@ -3,17 +3,5 @@ module.exports = function( api ) {
 
 	return {
 		presets: [ '@wordpress/babel-preset-default' ],
-		env: {
-			production: {
-				plugins: [
-					[
-						'@wordpress/babel-plugin-makepot',
```

Comment:
Arguably I wonder if it's still necessary to include `@wordpress/babel-plugin-makepot` as a dependency in `package.json`, since it's no longer referenced at all anywhere in the project. But I think it's probably simpler / non-harmful to just have `package.json` include _every single package_ we distribute.